src/introducer.py

```diff
@@ -38,7 +37,10 @@ async def _vetting_loop(self):
                 return
             try:
                 log.info("Vetting random peers.")
-                rawpeers = self.global_connections.peers.get_peers(
+                if self.global_connections.introducer_peers is None:
+                    await asyncio.sleep(3)
```

Comment:
Where does `3` come from? What are we really waiting for? Consider using a `Future` or `aiter`.